index3-1.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>課題❶-3</title>
+</head>
+<body>
+    <script>
+
+        let task = ['=====================',
+                       '現在持っているタスク一覧',
+                       '====================='];
+        let tasklist = ['掃除','買い物','散歩'];
+        
+        console.log(task[0]);
```

Comment:
せっかく配列に格納してあるので、以下の様にループさせてもいいのではないでしょうか？
```suggestion
        task.forEach(t => {
            console.log(t);
        });
```